src/pages/patterns/loaders.mdx

```diff
@@ -0,0 +1,249 @@
+before a page is loaded to assure a user that the content is on its way. Use spinners for pages and screens or within
+components such as tables. Center the spinner in the container or content you’re applying it to. Spinner is also used
+as an alternative loader to Skeleton in situations where the content can't be easily estimated but it's still
+important to provide a sense of progress.
+
+![Example of Spinner loader for unpredictable content](figma:patterns-loaders-spinner.png)
+
+### Completing action
+
+The <a href="/components/dots">Dots loader</a> communicates ongoing activity after a user takes an action. It tells
+them that something is taking place. Dots loader is used for buttons, search inputs, or other small components to
+indicate these components are doing something.
+
+Use the loader for actions where it might take several seconds for the process to complete.
+
+![Example of dot loaders when completing an action](figma:patterns-loaders-completing-action.png)
+
+1. <a href="/components/button#type">Primary buttons</a> are common candidates for using this loader
+1. <a href="/components/icon-button">Icon buttons</a> are less likely to use this loader but still necessary
+   if the user needs to be informed of the action taking place
+
+Use the loader for search inputs if it might take several seconds to perform the query.
+
+![Example of dot loaders during search](figma:patterns-loaders-search-inputs.png)
+
+Use the loader for <a href="/components/menu">Menus</a> where it might take several seconds to show information.
+
+![Example of dot loaders in a menu](figma:patterns-loaders-menu.png)
+
+### Progress
+
+A <a href="/components/progress">Progress loader</a> communicates progress from 0 to 100% when downloading or
+uploading content. The most common use for a progress loader is in the <a href="/components/file-upload">File upload</a>
+component. Descriptive text is displayed to help clarify
+what is in progress.
+
+![Example of progress loaders in a File upload](figma:patterns-loaders-progress.png)
+
+### Typing
+
+The <a href="/components/inline">Inline loader</a> is used as a typing indicator. This signals the other party is
+present and preparing a reply. The loader should be centered in the message bubble with other content
+appearing above or below.
+
+![Example of Inline loader when typing in chat](figma:patterns-loaders-typing.png)
+
+## Localization
+
+### Right-to-left languages
+
+Some loaders are direction-agnostic, which means that they are always positioned in the same place within the context,
+and use the same direction for any animation elements. Dots and Spinner are examples of that.
+
+Skeleton, Progress and Inline change direction for right-to-left languages. Skeleton animation and layout should change
+direction. Progress loader should fill from the right (0) to the left (100%) side. Inline animation should change direction.
+
+## Accessibility
```

Comment:
Is it worth making a note here about accessible animations? 🤔

Reply:
@jgorfine-zendesk do you have any good sources on that?

Reply:
@Francois-Esquire — yes! I don't know if we'll want to reference [this Confluence page directly](https://zendesk.atlassian.net/l/cp/92enuYtQ), but maybe we can pluck out the best info & links from there.

Reply:
Since this page already worked its way through design, we'll punt on this for now. Thanks.